### PR TITLE
[feature](multi-catalog) add specified_database_list PROPERTY for jdbc/hms/iceberg catalog

### PR DESCRIPTION
**Pull request by nextdreamblue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

add specified_database_list PROPERTY for jdbc catalog, user can use many database specified by jdbc catalog

## Checklist(Required)

* [ ] Does it affect the original behavior
* [x] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nextdreamblue** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.4 seconds
 stream load tsv:          455 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         39 seconds loaded 2358488459 Bytes, about 57 MB/s
 stream load orc:          73 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230315034247_clickbench_pr_114378.html

**Comment by nextdreamblue** [comment]
run p0

File: docs/en/docs/lakehouse/multi-catalog/jdbc.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Currently, Jdbc Catalog only support to use 5.x version of JDBC jar package to c
 | `driver_class ` | Yes             |               | JDBC Driver Class                                  |
 | `only_specified_database` | No             |     "false"          | Whether only the database specified to be synchronized.                                  |
 | `lower_case_table_names` | No             |     "false"          | Whether to synchronize jdbc external data source table names in lower case. |
+| `specified_database_list` | No             |     ""          | When only_specified_database=true，only synchronize the specified databases. split with ','|
```
Comment (morningman) [inline docs/en/docs/lakehouse/multi-catalog/jdbc.md:170]: maybe we can merge this with only_specilfied_database？

Comment (nextdreamblue) [reply docs/en/docs/lakehouse/multi-catalog/jdbc.md:170]: 是的，我做的时候就想这俩参数合并到一起，但是为了兼容已经使用 only_specified_database 这个参数的catalog，只能新增这个参数了
其他catalog可以只需要配置一个参数即可，specified_database_list来匹配指定的db

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run p1

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run clickbench

**Comment by nextdreamblue** [comment]
run clickbench

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run clickbench p0

**Comment by nextdreamblue** [comment]
run clickbench p0

**Comment by nextdreamblue** [comment]
run clickbench p0

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run buildlall

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run feut

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run p0

**Review by morningman — COMMENTED** [review]
I think `only_specified_database` this could be a common property for all kinds of catalog?

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run feut

File: fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java [hunk 2]
```diff
@@ -306,6 +318,20 @@ public boolean isTableExist(String dbName, String tableName) {
         }
     }
 
+    public void getSpecifiedDatabaseMap(String specifiedDatabaseList) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java:321]: Why is here another `getSpecifiedDatabaseMap()` method?
I think we unify this logic for all kinds of catalog

Comment (nextdreamblue) [reply fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java:321]: it is better, let me fix it

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**